extensions/tc-no-generator/package.json

```diff
@@ -0,0 +1,48 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "tc-no-generator",
+  "title": "TC NO Generator",
+  "description": "tc no üretip clipboarda kopyalar",
+  "icon": "cmd-icon.png",
+  "author": "denizyoldas",
+  "categories": [
+    "Data"
+  ],
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "tc-no-generate",
+      "title": "Generate Tc No",
+      "subtitle": "Tc No",
+      "description": "tc no üretip kopyalar",
+      "mode": "no-view"
+    },
+    {
+      "name": "last-tc",
+      "title": "Copy last Tc No",
+      "subtitle": "TC No",
+      "description": "son üretilen TcNo kopyalar",
```

Comment:
At the moment, Raycast doesn't support localization and only supports US English. Therefore, please avoid introducing your custom way to localize your extension.